_episodes/11-esmvalcoreapi.md

```diff
@@ -0,0 +1,511 @@
+---
+title: "ESMValCore API in a Jupyter notebook"
```

Comment:
In the tutorial so far, we have not distinguished between ESMvalCore and ESMValTool. Also we have not referred to the ESMValCore API as such, so perhaps we should write a line or two to orient the user.

Reply:
I have extended the title and added more to the start of the episode